src/registry_schemas/example_data/schema_data.py

```diff
@@ -85,6 +85,165 @@
     'deliveryInstructions': 'some delivery instructions'
 }
 
+COURT_ORDER = {
+    'fileNumber': '#1234-5678/90',
+    'orderDate': '2021-01-30T09:56:01+08:00',
+    'effectOfOrder': 'planOfArrangement',
+    'orderDetails': 'A note about order',
+    'fileKey': '011e332d-1b8e-4218-8710-ad8ac1fbc592.pdf'
+}
+
+FOREIGN_JURISDICTION = {
+    'country': 'CA',
+    'region': 'AB'
+}
```

Comment:
Note that region is optional (only used if country is Canada).

Also note that region could be "Federal".

Reply:
^^ this is already captured in the foreign jurisdiction schema :)